src/lib/numbers.ts

```diff
@@ -16,6 +16,30 @@ export function expandDecimals(n: BigNumberish, decimals: number): BigNumber {
   return bigNumberify(n).mul(bigNumberify(10).pow(decimals));
 }
 
+const MAX_EXCEEDING_THRESHOLD = "1000000000";
+const MIN_EXCEEDING_THRESHOLD = "0.01";
+
+function checkPriceThresholdExceeding(value: BigNumber, tokenDecimals: number) {
+  const maxExceeding = expandDecimals(MAX_EXCEEDING_THRESHOLD, tokenDecimals);
+  const minExceeding = ethers.utils.parseUnits(MIN_EXCEEDING_THRESHOLD, tokenDecimals);
+
+  if (!value || value.eq(0)) return;
```

Comment:
mm, maybe return original value and empty "" symbol if value doesn't exceed threshold? to avoid undefined and duplicating conditional using like exceedingInfo?.symbol ?? ""